client/src/featureform/register.py

```
@@ -4874,6 +4880,111 @@ def apply(self, asynchronous=False, verbose=False):
                 set_run("")
             clear_state()
 
+    DeletableResource = Union[
+        FeatureColumnResource,
+        SubscriptableTransformation,
+        LabelColumnResource,
+        TrainingSetVariant,
```

Ruff (0.8.2)

`TrainingSetVariant` may be undefined, or defined from star imports

(F405)